frontend/src/metabase/visualizations/components/settings/ChartSettingMessage.tsx

```diff
@@ -1,20 +0,0 @@
-import type { PropsWithChildren } from "react";
-
-import { Center, useMantineTheme } from "metabase/ui";
-
-export const ChartSettingMessage = ({ children }: PropsWithChildren) => {
```

Comment:
@oisincoveney can we safely remove it?

Reply:
I don't think so - we use it in:

```
frontend/src/metabase/visualizations/components/settings/ChartSettingSeriesOrder.tsx:18:10:..
frontend/src/metabase/visualizations/components/settings/ChartSettingSeriesOrder.tsx:272:10..
frontend/src/metabase/visualizations/components/settings/ChartSettingSeriesOrder.tsx:272:53..
frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx:12:1.. frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx:118:..
frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx:118:.
```

At least according to the `get references` command on neovim

Reply:
isn't it come from 

```
import {
  ChartSettingMessage,
  ChartSettingOrderedSimpleRoot,
} from "./ChartSettingOrderedSimple.styled";
```

But the name is the same. I'm asking you as you've recently touched it

Reply:
Ah, I'm on the branch that's actually using that component ([this one](https://github.com/metabase/metabase/pull/51332/files)) . That will delete `ChartSettingOrderedSimple.styled` and use this component

Reply:
Let's merge this PR, and then I'll put it back in my PR where it belongs

Reply:
I can revert this file, no problem